src/rapidfuzz/process_cpp_impl.pyi

```diff
@@ -16,79 +16,79 @@ from typing import (
 from rapidfuzz.fuzz import WRatio, ratio
 
 _StringType = Sequence[Hashable]
-_AnyStringType = TypeVar("_AnyStringType", bound=_StringType)
```

Comment:
I simply forgot to delete this file. Both `process_cpp_impl.pyi` and `metrics_cpp.pyi` are going to get removed.